src/clients/api/queries/useGetUserMarketInfo.ts

```diff
@@ -0,0 +1,305 @@
+import { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+
+import { IGetMarketsOutput } from 'clients/api/queries/getMarkets';
+import { TREASURY_ADDRESS } from 'config';
+import { useVaiUser } from 'hooks/useVaiUser';
+import { Asset, Market } from 'types';
+import { indexBy, convertCoinsToWei } from 'utilities/common';
+import { calculateCollateralValue, getVBepToken, getToken } from 'utilities';
+import { VBEP_TOKENS, TOKENS } from 'constants/tokens';
+import {
+  useGetMarkets,
+  useGetAssetsInAccount,
+  useGetVTokenBalancesAll,
+  IGetVTokenBalancesAllOutput,
+} from 'clients/api';
+
+export interface IData {
+  assets: Asset[];
+  userTotalBorrowLimitCents: BigNumber;
+  userTotalBorrowBalanceCents: BigNumber;
+  userTotalSupplyBalanceCents: BigNumber;
+  treasuryTotalSupplyUsdBalanceCents: BigNumber;
+  treasuryTotalAvailableLiquidityUsdBalanceCents: BigNumber;
+  treasuryTotalBorrowUsdBalanceCents: BigNumber;
+  treasuryTotalUsdBalanceCents: BigNumber;
+  totalXvsDistributedWei: BigNumber;
+  dailyVenus: BigNumber;
+}
+
+export interface UseGetUserMarketInfoOutput {
+  isLoading: boolean;
+  data: IData;
+}
+
+const vTokenAddresses: string[] = Object.values(VBEP_TOKENS).reduce(
+  (acc, item) => (item.address ? [...acc, item.address] : acc),
+  [],
+);
+
+// TODO: decouple, this hook handles too many things (see https://app.clickup.com/t/2d4rfx6)
```

Comment:
nit: This name doesn't completely make  sense because there is not `getUserMarketInfo` query. This could be improved when it is refactored per the todo

Reply:
We haven't really discussed this before, but I don't think using `useGetSomething` needs to mean that it calls a function that is called `getSomething`. I've used the prefix `useGet` to indicate it's a hook (`use`) and that it's a query ('Get')